google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanelTest.java

```diff
@@ -67,7 +67,9 @@
 
   private GcsBucketContentEditorPanel editorPanel;
 
-  @TestService @Mock private GoogleLoginService loginService;
+  @TestService
+  @Mock
+  private IntegratedGoogleLoginService loginService;
```

Comment:
nit: Can you keep these all on one line? If not, you can move this into its own block below so the single-annotation fields are together for easier readability.

Reply:
fixed

Reply:
is this fixed? still looks like the @Mock is on its own line